dlrover/python/unified/backend/elastic/master.py

```diff
@@ -12,42 +12,41 @@
 # limitations under the License.
 
 
-import asyncio
-
-import ray
-
 from dlrover.python.unified.common.actor_base import ActorBase
+from dlrover.python.unified.common.enums import ExecutionResult
 from dlrover.python.unified.controller.api import PrimeMasterApi
-from dlrover.python.unified.util.async_helper import unsafe_run_blocking
+from dlrover.python.unified.util.decorators import log_execution
 
 from .manager import ElasticManager
 
 
 class ElasticMaster(ActorBase):
-    def _setup(self):
+    def setup(self):
         workers = PrimeMasterApi.get_workers_by_role(self.actor_info.role)
         self.manager = ElasticManager(workers)
-        if ray.get_runtime_context().was_current_actor_reconstructed:
-            unsafe_run_blocking(
-                self.manager._recover_running(
-                    main_loop=asyncio.get_running_loop()
-                )
-            )
 
     # Lifecycle Hooks
 
-    def get_stage(self):
-        return self.manager.stage
-
     async def check_workers(self):
         await self.manager.check_workers()
 
     async def start(self):
-        await self.manager.start()
+        with log_execution("ElasticMaster start"):
+            await self.manager.start()
+        # ElasticMaster is Servicer, not affect the job result
+        PrimeMasterApi.report_execution_result(
```

Comment:
Consistent with the discussion above, i suggest that the servicer should not be passed as a parameter here. Instead, the Elastic Master, through its own attribute fields, should execute the corresponding independent logic as expected when report_execution_result is called.